src/workerd/api/http.c++

```diff
@@ -1210,6 +1210,14 @@ jsg::Ref<AbortSignal> Request::getThisSignal(jsg::Lock& js) {
   return newSignal;
 }
 
+kj::Maybe<jsg::Ref<AbortSignal>> Request::getSignalForSubrequest(jsg::Lock& js) {
+  if (FeatureFlags::get(js).getEnableRequestSignal()) {
```

Comment:
Nit: some additional documentation here to provide context would be helpful... I know there's comments in the http.h but having it here with the code as well would be ideal